src/CONST.ts

```diff
@@ -543,6 +543,7 @@ const CONST = {
                     INDIVIDUAL_BUDGET_NOTIFICATION: 'POLICYCHANGELOG_INDIVIDUAL_BUDGET_NOTIFICATION',
                     INVITE_TO_ROOM: 'POLICYCHANGELOG_INVITETOROOM',
                     REMOVE_FROM_ROOM: 'POLICYCHANGELOG_REMOVEFROMROOM',
+                    REPLACE_CATEGORIES: 'POLICYCHANGELOG_REPLACE_CATEGORIES',
```

Comment:
Does this constant have any uses?

Reply:
It is used here with the spread operator, if this constant is not here this action type is not shown in the chat.

https://github.com/Expensify/App/blob/aa1d87caddb6831061f3750bd78f77b42603e5e3/src/pages/home/report/ReportActionItem.js#L445

Reply:
Oh, it's tricky. So we never refer to it TypeScript-wise. Thank you for clarification!